tealeaf/TeaLeafIOS.xcodeproj/project.pbxproj

```diff
@@ -1373,7 +1362,6 @@
 			isa = PBXGroup;
 			children = (
 				E25AB2641F97557200069969 /* Stickers.xcassets */,
-				E25AB2661F97557200069969 /* Info.plist */,
```

Comment:
is this change okay ?

Reply:
Xcode itself modified the file on opening.I am not sure whether we need it or not.